oak-jcr/src/test/java/org/apache/jackrabbit/oak/jcr/binary/BinaryAccessTest.java

```diff
@@ -111,6 +123,17 @@ private static URI expectedUploadURI() {
         return toURI(UPLOAD_URL);
     }
 
+    private static String DEFAULT_CDN_UPLOAD_URL = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/oak-indexing/jackrabbit-oak/pullRequest?prid=5115210)